Changeloghtml (#983)
dotnet/upgrade-assistant: README.md

Upgrade-assistant is installed as a NuGet package, so invalid or authenticated sources in [NuGet configuration](https://docs.microsoft.com/nuget/consume-packages/configuring-nuget-behavior) can cause installation problems.

> To try the latest (and likely less stable) versions of the tool, CI builds are available on the dotnet-tools NuGet feed and can be installed with

Reviewer: ```suggestion
To try the latest (and likely less stable) versions of the tool, CI builds are available on the dotnet-tools NuGet feed and can be installed with 
```

> To try the latest (and likely less stable) versions of the tool, CI builds are available on the dotnet-tools NuGet feed and can be installed with

Reviewer: ```suggestion
To try the latest (and likely less stable) versions of the tool, CI builds are available on the dotnet-tools NuGet feed and can be installed with 
```

                                                 Preview TFM. See
                                                 https://dotnet.microsoft.com/platform/support/policy/dotnet-core for
                                                 details for what these mean.

>                                                                     --format                                        Specify what format desired. Currently supports html and sarif. 

Reviewer: The formatting of this line is super odd, can you align it with other lines above? Also use the preview tab above to see how the text will look like so you can get a better idea how it looks visually.

  --version                                      Show version information
  -?, -h, --help                                 Show help and usage information
```
The output of the analyze command is a report in SARIF format. SARIF is based on JSON and can be viewed using the following viewers:

> - Any text editor.

Reviewer: ```suggestion
- Any text editor. 
```

- [VS extension for SARIF Viewer](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer) for a richer experience.

> Sample of the report in VS SARIF Viewer : ![Analysis Report](/docs/images/AnalysisReport.PNG)

Reviewer: Could you add details about HTML Viewer here? similar to how we are taking about SARIF log, with a screenshot of a sample HTML View.

                                                 Preview TFM. See
                                                 https://dotnet.microsoft.com/platform/support/policy/dotnet-core for
                                                 details for what these mean.

>   --format  <HTML>                                     Specify the format in which the analysis report will be generated. Currently supports html other than the default SARIF format.

Reviewer: ```suggestion
  --format  <HTML>                               Specify the format in which the analysis report will be generated. Currently supports html other than the default SARIF format.
```

  --version                                      Show version information
  -?, -h, --help                                 Show help and usage information
```
The output of the analyze command is a report in SARIF format. SARIF is based on JSON and can be viewed using the following viewers:

> - Any text editor.

Reviewer: ```suggestion
- Any text editor. 
```

                                                 Preview TFM. See
                                                 https://dotnet.microsoft.com/platform/support/policy/dotnet-core for
                                                 details for what these mean.

>   --format                                       Specify what format desired. Currently supports html and sarif. 

Reviewer: ```suggestion
  --format  <HTML>                                     Specify the format in which the analysis report will be generated. Currently supports html other than the default SARIF format. 
```

Take a look at the high level overview of the roadmap for this tool and the journey to upgrade your apps from .NET Framework to the latets version of .NET (current, LTS, or preview) and beyond in the [roadmap](docs/roadmap.md).

## Engage, Contribute and Give Feedback

> Some of the best ways to contribute are to use the tool to upgrade your apps to the latest version of .NET (current, LTS, or preview), file issues for feature-requests or bugs, join in design conversations, and make pull-requests.

Reviewer: ```suggestion
Some of the best ways to contribute are to use the tool to upgrade your apps to the latest version of .NET (current, LTS, or preview), file issues for feature-requests or bugs, join in design conversations, and make pull-requests. 
```

> 

Reviewer: looks like you formatted some of the previous changes, is that needed?

Author: An accidental erasure during editing

Reviewer: can you revert the accidental changes? basically I'm looking for just the change to add the HTMLView content here. The rest of the changes here are making this PR noisy

Reviewer: Remove the accidental formatting changes here.

Take a look at the high level overview of the roadmap for this tool and the journey to upgrade your apps from .NET Framework to the latets version of .NET (current, LTS, or preview) and beyond in the [roadmap](docs/roadmap.md).

## Engage, Contribute and Give Feedback

> Some of the best ways to contribute are to use the tool to upgrade your apps to the latest version of .NET (current, LTS, or preview), file issues for feature-requests or bugs, join in design conversations, and make pull-requests.

Reviewer: ```suggestion
Some of the best ways to contribute are to use the tool to upgrade your apps to the latest version of .NET (current, LTS, or preview), file issues for feature-requests or bugs, join in design conversations, and make pull-requests. 
```
